addons/account_peppol/views/account_journal_dashboard_view.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="account_journal_dashboard_kanban_view" model="ir.ui.view">
+        <field name="name">account.journal.dashboard.kanban</field>
+        <field name="model">account.journal</field>
+        <field name="inherit_id" ref="account.account_journal_dashboard_kanban_view" />
+        <field name="arch" type="xml">
+            <data>
+                <xpath expr="//kanban" position="inside">
+                    <field name="is_peppol_journal" invisible="1"/>
+                    <field name="account_peppol_proxy_state" invisible="1"/>
+                </xpath>
+
+                <xpath expr="//t[@id='account.JournalBodySalePurchase']//div" position="inside">
+                    <t t-set="peppol_state" t-value="record.account_peppol_proxy_state.raw_value"/>
+                    <t t-if="peppol_state == 'active'">
+                        <t t-if="journal_type == 'sale'">
+                            <a type="object" name="peppol_get_message_status" groups="account.group_account_invoice">Update Peppol Status</a>
+                        </t>
+                        <t t-if="journal_type == 'purchase'">
+                            <t t-set="show_fetch_action" t-value="record.is_peppol_journal.raw_value"/>
+                            <t t-if="show_fetch_action">
```

Comment:
Sam than what @aliyatastemirova told you above. You can pass directly your `t-value` in the` t-if `:)